docs/status/README.md

```diff
@@ -1,114 +1,112 @@
 
-You need to click on the **Add checklist** button, and the first item of the checklist appears.
+![Add checklist](../img/getting-started/add_checklist.png)
 
-![add checklist](../img/getting-started/add_checklist.png)
+Hit **Enter** or click the button again to add more checklist items, and give each one a name.
 
-Type your comments, hit the **Enter** key to add another line to your checklist, or click the **Add Checklist** button again.
+![Checklist](../img/getting-started/checklist_detailed.png)
 
-![checklist](../img/getting-started/checklist_detailed.png)
-
-In addition to your comment, you can attach a file as a picture. Click on the **Add attachment** button.
+To attach a file as a picture, click on the **Add attachment** button.
 
 ![Add Attachment Simple](../img/getting-started/attachment_snapshot.png)
 
-Select a **file from your hard drive** or past a **screenshot** here.
+Select a **file from your hard drive** or paste a **screenshot**.
 
 ![Attachment snapshot](../img/getting-started/add_attachmen_simplet.png)
 
-
 Validate the attachment by clicking on **Add file as attachment**.
 
-
-Then select the **Status** that you need on the dropdown menu. 
+Next, select the **Status** you wish to change the task to from the dropdown menu.
 
 ![Status List](../img/getting-started/status_list.png)
 
-Then click the **Post** button to post it to the world.
+Finally, click the **Post** button to submit your comment and status update.
 
+## Changing Statuses In Bulk
 
-## Change Status per Batch
+You can also change the status for multiple tasks at once.
 
-Alternatively, you can do it per **batch**.
-
-You can select multiple shots or assets by pressing the **ctrl** or
-**shift** key.
+Select multiple shots or assets by pressing **Ctrl / Cmd** or **Shift** and select the tasks you wish to include.
 
 Then, go to the action menu's **Change status** section.
 
-![action menu status](../img/getting-started/blue_menu_status.png)
+![Action menu status](../img/getting-started/blue_menu_status.png)
 
-Choose the new status for your selected tasks (1). You can also add a comment for all the selected tasks (2)
-You validate the new status with the **Confirm** button.
+Choose the new status for your selected tasks (1). You can also add a comment for all the selected tasks (2). Save the new status with the **Confirm** button.
 
 ![Change status per batch](../img/getting-started/change_status_batch.png)
+
 ![Metadata Column Sort By](../img/getting-started/custom_column_sortby.png)
 
-The shots are not sorted by name but by the metadata column information.
+<!-- Everything from this point onward until the next comment doesn't make sense here. Previously we were talking about changing statuses in bulk, but now we're talking about sorting tasks and editing metadata columns. Is this supposed to be here? At the very least we should give it it's own heading or move it somewhere else -->
```

Comment:
It seems there is a problem. You should talk about it with Gwen.

Reply:
Confirmed it was an error, this has been removed